CONTRIBUTING.md

```diff
@@ -1,57 +1,46 @@
 # Contributing
 
-Thanks for your interest in contributing to this project! PRs and issues are welcome, but please keep in mind that this project is still and alpha and the the implementation details and API will most likely change between now and a stable release.
+Thanks for your interest in contributing to this project! PRs and issues are welcome.
 
 For larger changes, please first make an [RFC issue](https://github.com/railwayapp/nixpacks/issues). You can follow along with the roadmap in the [GitHub project](https://github.com/railwayapp/nixpacks/projects/1).
```

Comment:
I think we can remove this roadmap, it's no longer in use.

```suggestion
For larger changes, please start a discussion by [creating an issue](https://github.com/railwayapp/nixpacks/issues/new?assignees=&labels=&template=feature_request.yml&title=Nixpacks+Feature+Request).
```

Thoughts on ^?